sdk/digitaltwins/digital-twins/src/generated/operations/digitalTwinModels.ts

```diff
@@ -169,7 +169,7 @@ const addOperationSpec: coreHttp.OperationSpec = {
   path: "/models",
   httpMethod: "POST",
   responses: {
-    200: {
+    201: {
```

Comment:
Was this change intended?

Reply:
Yes, this is generated code using the latest swagger

---
In reply to: [483755756](https://github.com/Azure/azure-sdk-for-js/pull/11045#discussion_r483755756) [](ancestors = 483755756)